pysatNASA/instruments/methods/_cdf.py

```diff
@@ -0,0 +1,442 @@
+            epoch dependency data
+        """
+
+        self._dependencies[self._filename + x_axis_var] = x_axis_data
+
+    def set_epoch(self, x_axis_var):
+        """Stores epoch dependency
+
+        Parameters
+        ----------
+        x_axis_var : str
+            name of variable
+
+        """
+
+        data_type_description \
+            = self._cdf_file.varinq(x_axis_var)['Data_Type_Description']
+
+        center_measurement = self._center_measurement
+        cdf_file = self._cdf_file
+        if self.get_dependency(x_axis_var) is None:
+            delta_plus_var = 0.0
+            delta_minus_var = 0.0
+            has_plus_minus = [False, False]
+
+            xdata = cdf_file.varget(x_axis_var)
+            epoch_var_atts = cdf_file.varattsget(x_axis_var)
+
+            # Check for DELTA_PLUS_VAR/DELTA_MINUS_VAR attributes
+            if center_measurement:
+                if 'DELTA_PLUS_VAR' in epoch_var_atts:
+                    delta_plus_var = cdf_file.varget(
+                        epoch_var_atts['DELTA_PLUS_VAR'])
+                    delta_plus_var_att = cdf_file.varattsget(
+                        epoch_var_atts['DELTA_PLUS_VAR'])
+                    has_plus_minus[0] = True
+
+                    # Check if a conversion to seconds is required
+                    if 'SI_CONVERSION' in delta_plus_var_att:
+                        si_conv = delta_plus_var_att['SI_CONVERSION']
+                        delta_plus_var = delta_plus_var.astype(float) \
+                            * np.float(si_conv.split('>')[0])
+                    elif 'SI_CONV' in delta_plus_var_att:
+                        si_conv = delta_plus_var_att['SI_CONV']
+                        delta_plus_var = delta_plus_var.astype(float) \
+                            * np.float(si_conv.split('>')[0])
+
+                if 'DELTA_MINUS_VAR' in epoch_var_atts:
+                    delta_minus_var = cdf_file.varget(
+                        epoch_var_atts['DELTA_MINUS_VAR'])
+                    delta_minus_var_att = cdf_file.varattsget(
+                        epoch_var_atts['DELTA_MINUS_VAR'])
+                    has_plus_minus[1] = True
+
+                    # Check if a conversion to seconds is required
+                    if 'SI_CONVERSION' in delta_minus_var_att:
+                        si_conv = delta_minus_var_att['SI_CONVERSION']
+                        delta_minus_var = \
+                            delta_minus_var.astype(float) \
+                            * np.float(si_conv.split('>')[0])
+                    elif 'SI_CONV' in delta_minus_var_att:
+                        si_conv = delta_minus_var_att['SI_CONV']
+                        delta_minus_var = \
+                            delta_minus_var.astype(float) \
+                            * np.float(si_conv.split('>')[0])
+
+            if ('CDF_TIME' in data_type_description) or \
+                    ('CDF_EPOCH' in data_type_description):
```

Comment:
Recommend breaking before operator
```suggestion
            if ('CDF_TIME' in data_type_description) \
                    or ('CDF_EPOCH' in data_type_description):
```